docker-compose-e2e.account.yaml

```diff
@@ -32,13 +32,16 @@ services:
       - e2e
 
   frequency:
+    image: dsnp/instant-seal-node-with-deployed-schemas:latest
+    # We need to specify the platform because it's the only image
+    # built by Frequency at the moment, and auto-pull won't work otherwise
+    platform: linux/amd64
+    # SEALING_MODE set to insstant for e2e testing
+    environment:
+      - SEALING_MODE=interval
+      - SEALING_INTERVAL=1
```

Comment:
Shouldn't need to add this here; don't we always layer this file in *after* the main `docker-compose.yaml`, which already specifies this?